be/src/cloud/cloud_tablet.h

```diff
@@ -145,6 +145,9 @@ class CloudTablet final : public BaseTablet {
         _last_base_compaction_schedule_millis = millis;
     }
 
+    int64_t alter_version() const { return _alter_version; }
+    void set_alter_version(int64_t alter_version) { _alter_version = alter_version; }
```

Comment:
When to reset alter version = -1

Reply:
BE restart or quit from process_alter_tablet. In process_alter_tablet, there have a defer to change it.